.github/workflows/osx_debug.yml

```diff
@@ -32,14 +32,20 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  osx_11_lto:
+  osx_debug:
     # Run on push to the 'master' and release branches of tarantool/tarantool
     # or on pull request if the 'full-ci' label is set.
     if: github.repository == 'tarantool/tarantool' &&
         ( github.event_name != 'pull_request' ||
           contains(github.event.pull_request.labels.*.name, 'full-ci') )
 
-    runs-on: [ macos-11-self-hosted, x86_64 ]
+    runs-on: [ 'macos-${{ matrix.version }}-self-hosted', '${{ matrix.arch }}' ]
```

Comment:
you don't need quotes here

Reply:
Actually, I need them. Otherwise, I get a syntax error.

Reply:
oh, okay, thanks for checking